packages/formation-react/src/components/Telephone/Telephone.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Telephone
+name: Telephone
+tags: telephone, component
+---
+
+import Telephone, { CONTACTS, PATTERNS } from './Telephone';
+
+### Code:
+```javascript
+import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/formation-react/ProgressBar'
```

Comment:
Is this the right file to import from? It's different than the unit test, so figured it was worth an ask.

Reply:
Oh, good catch!